meson.build

```diff
@@ -17,7 +17,9 @@ endif
 
 thread_dep = dependency('threads')
 kiwixlib_dep = dependency('kiwix', version:'>=13.0.0', static:static_linkage)
-libzim_dep = dependency('libzim', version:'>=8.1.0', static:static_linkage)
+kiwixlib_dep = dependency('kiwix', version:'<14.0.0', static:static_linkage)
```

Comment:
Maybe check the found dependency version (with https://mesonbuild.com/Reference-manual_returned_dep.html#depversion) instead of searching for a new dependency again.

Reply:
The second check relies on the cache of meson. I understand your proposal, but not convinced this is just better than reasking meson with new parameter so he can answer based on its own cache. A little bit less elegant on the principle but better in term of clarity and maintainance. But feel free to propose an alternative on this PR.

Reply:
Any feedback, should I just merge as it?